### PR TITLE
CSS image-orientation: <angle> & flip obsolete

### PR DESCRIPTION
**Pull request by estelle**
https://bugzilla.mozilla.org/show_bug.cgi?id=1473450

### CODE REVIEW

**Comment by Elchi3** [comment]
So, this PR changes a few things at once and I don't know if all of them are intended.
The data structure right now looks like this:

- `image-orientation`:
  - `from-image`
  - `flip`

After this PR, the structure is changed to:

- `image-orientation`
  - `from-image` (with a description saying "<code>flip</code> &amp; <code>&lt;angle&gt;</code>" which doesn't seem to belong to `from-image` at all).

I would have expect everything set to deprecated and this structure:
- `image-orientation`:
  - `from-image` --> remains as is, still implemented in Firefox
  - `flip` --> becomes "<code>flip</code> &amp; <code>&lt;angle&gt;</code>" and is marked as removed

What do you think, @estelle?

**Comment by estelle** [comment]
There are only 2 values: none and from-image, so "basic support" means from-image is supported. I don't think there's a reason to list out that value since that is what basic support means.
image-orientation:
`from-image` -->becomes `basic-support`
flip --> becomes "flip & <angle>" and is marked as removed - exactly

**Comment by Elchi3** [comment]
> so "basic support" means from-image is supported. I don't think there's a reason to list out that value since that is what basic support means.

Ahhh, gotcha! This is the bit that explains it. Thank you!

File: css/properties/image-orientation.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
         },
         "from-image": {
```
Comment (Elchi3) [inline css/properties/image-orientation.json]: This is not about from-image then anymore, so the identifier should probably be changed to maybe "flip_and_angle".

File: css/properties/image-orientation.json [hunk 2]
```diff
@@ -98,58 +100,7 @@
             "status": {
               "experimental": true,
```
Comment (Elchi3) [inline css/properties/image-orientation.json]: Probably not an experiment anymore, now that it is removed. false?

File: css/properties/image-orientation.json (hunk 1 above)
Comment (estelle) [reply css/properties/image-orientation.json]: good catch. thanks.

**Pull request merged**